packages/tokens/src/products/shared/app-header.json

```diff
@@ -0,0 +1,18 @@
+{
+  "app-header": {
+    "height": {
+      "value": "60",
+      "type": "size"
+    },
+    "home-link": {
+      "padding": {
```

Comment:
Consider an alternative approach: using two variables, one for the overall `home-link` size and one for the `logo` size, and infer the internal padding from these two.

Reply:
btw, I don't see the `--token-app-header-home-link-logo-size` used anywhere? maybe it's still a WIP?

Reply:
Ok I'll see. I think I forgot to add the property for the logo-size in the styles. I will update. (I was trying to follow some of the pattern from the SideNav HomeLink)

Reply:
Alright, I refactored as you suggested.